tests/foreman/ui_airgun/test_computeresource.py

```diff
@@ -206,3 +228,36 @@ def test_positive_resource_vm_power_management(
             session.computeresource.vm_poweron(name, rhev_data['vm_name'])
         assert session.computeresource.vm_status(
             name, rhev_data['vm_name']) is not status
+
+
+@tier2
+@parametrize('provider', _provider_parameteres)
+def test_positive_create_edit_desc_delete_resource(session, module_org, module_loc, provider):
```

Comment:
`test_positive_end_to_end`

Reply:
Maybe `@tier3` would be more appropriate?

Reply:
in all other modules, we called real CRUD scenario (that doing all 4 operations and with many different parameters) as end-to-end, because it is actually verify component in isolated state from all possible sides. And `real world scenario` is really close to tier3 ideology =)
I know that is not so perfect explanation and idea, but I would prefer to keep consistency rather than correctness and especially avoid such long test names